ethers-core/src/types/transaction/eip2718.rs

```diff
@@ -0,0 +1,84 @@
+use super::{eip1559::Eip1559TransactionRequest, eip2930::Eip2930TransactionRequest};
+use crate::{
+    types::{TransactionRequest, H256, U64},
+    utils::keccak256,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Debug)]
+#[serde(tag = "type")]
+pub enum TypedTransaction {
+    // 0x00
+    #[serde(rename = "0x00")]
+    Legacy(TransactionRequest),
+    // 0x01
+    #[serde(rename = "0x01")]
+    Eip2930(Eip2930TransactionRequest),
```

Comment:
Can I request a rename of these transaction names?

I prefer python's version of "AccessListTransactionRequest" and "DynamicFeeTransactionRequest" over these number named one.

Eip2930 and Eip1559 are not straightforward, it's hard to figure out the underlying meaning.

Reply:
A sample definition here: https://github.com/guanqun/ethers-rs/blob/feat/typed-transactions/ethers-core/src/types/typed_transaction.rs#L42

Reply:
Hey - we discussed it in the dev chat, I preferred the EIPxxx numbering, confirmed by @prestwich @roynalnaruto @mattsse, so we'll keep the EIPXXX numbering for tx types!